src/style/card__img.css

```diff
@@ -0,0 +1,5 @@
+.card__img {
```

Comment:
no need to create separate file, it is not bem block

Reply:
done